amgut/templates/animal_survey.html

```diff
@@ -1,241 +1,77 @@
 {% extends sitebase.html %}
+{% from wtforms import validators %}
+{% block head %}
+<import >
+<script type="text/javascript">
+    function clone_pet() {
```

Comment:
needed so you can add multiple other pets the pet in question lives with. Allows you to create a new dropdown if there is more than one.